.github/workflows/ci-disable-gtest.yml

```diff
@@ -5,7 +5,13 @@
 #   * Disables gtest.
 
 name: CI Disable GTest
-on: [push, pull_request]
+on:
+  push
+  pull_request:
+    paths:
+      - '**CMakeLists.txt**'
```

Comment:
The trailing '**' might not be necessary.

Reply:
Done